src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -87,6 +90,26 @@ export default class ProcessConverter {
     }
   }
 
+  private assignIncomingAndOutgoingIdsFromFlows(): void {
+    const fillShapeBpmnElementAttribute = (
```

Comment:
**question:** why defining this function here and not as a private method of the class?
This is not consistent with the rest of the code (we always create private methods in this case).
It may be a good reason, in that case, I suggest to document it as this is unclear to me

Reply:
As this is a technical method, only used within this context and I wasn't sure where to put it, I preferred to leave it here. But I am open to any ideas. ^^